packages/connector-local/package.json

```diff
@@ -7,6 +7,7 @@
     "timeout": "1m"
   },
   "dependencies": {
+    "@hint/parser-html": "^0.0.1",
```

Comment:
Note to self: Move this into a different commit.

Reply:
Honestly that should be a dev dependency as it's only used for types (specifically the `HTMLParse` event). The only functional tie `connector-local` has to the `parser-html` is via the `parse::html::end` event. I'll fix it (though that probably doesn't change your need to add even the dev dependency in a separate commit).

Reply:
I was referring a different commit, not anything about coding changing.

That is for the release process.